jekyll/_layouts/classic-docs.html

```diff
@@ -1,58 +1,138 @@
+        {% if page.page-type == "index" %}<h2>{{ page.title }}</h2>{% endif %}
+
+
+        <article>
+          <!-- Title and version badges: -->
+          <div>
+            {% if page.page-type != "index" and page.page-type != "homepage" %}<h1 id="{{ page.title | downcase | split: " " | join: "-" }}">{{ page.title }}</h1>{% endif %}
           </div>
-          {% for version in page.version %}
-          <span class="badge badge-primary">
-            <button class="server-version-badge" aria-describedby="tooltip">{{ version }}</button>
-            <!-- <img src="{{ site.baseurl }}/assets/img/docs/{{ version }}-badge.png" height="24">  -->
-          </span>
-          {% endfor %}
-        </div>
-        {% endif %}
-        {% if page.untranslated %}
-          {% include_cached translation_in_progress.html site=site page=page %}
-        {% endif %}
-        {{ content }}
-        {% if page.collection != 'cci1' and page.page-type != "index" and page.layout == "classic-docs" or page.layout == "enterprise" %}
-          {% include_cached doc-footer.html page=page site=site %}
+
+            {% if page.version %}
+            <div class="server-version-wrapper">
+              <div id="tooltip" role="tooltip" class="server-version-popover">
+                This document is applicable to the following:
+                <div id="arrow" data-popper-arrow></div>
+              </div>
+              {% for version in page.version %}
+              <span class="badge badge-primary">
+                <button class="server-version-badge" aria-describedby="tooltip">{{ version }}</button>
+                <!-- <img src="{{ site.baseurl }}/assets/img/docs/{{ version }}-badge.png" height="24">  -->
+              </span>
+              {% endfor %}
+            </div>
+            {% endif %}
+
+
+          {% if page.untranslated %}
+            {% include translation_in_progress.html %}
+          {% endif %}
+
+          {{ content }}
+
+          {% if page.collection != 'cci1' and page.page-type != "index" and page.layout == "classic-docs" or page.layout == "enterprise" %}
+            {% include doc-footer.html %}
+          {% endif %}
+
+        </article>
+
+      </div><!-- .article-container -->
+
+        {% if page.side_toc != "none" %}
+        <aside class="table-of-contents">
+          <h5 class="toc-heading">On this page </h5>
+          <div class="toc-indent">
+            {{ content | toc_only }}
+          </div>
+        </aside>
         {% endif %}
-      </article>
-    </div><!-- .article-container -->
-  </div><!-- .main-body -->
+    </div><!-- .sidebar-offset -->
+
+	{% include_localized global-footer.html %}
+	</div><!-- .main-body -->
 
-  {% include_localized global-footer.html %}
-  {% include_cached js-config.html site=site page=page %}
-  {% include_cached js-assets.html %}
+	{% include js-config.html %}
+	{% asset vendor.min.js %}
```

Comment:
Most (all?) of the scripts added here and below were extracted into the `js-assets.html` include that was removed on [this line](https://github.com/circleci/circleci-docs/pull/5396/files?diff=unified&w=1#diff-2843dc47d0bb4f5f2cb5899901cbab0f7b03f649308e7464e9861d7bd3420af5L55).